packages/forms-web-app/src/pages/project-search/utils/get-page-data.js

```diff
@@ -6,12 +6,17 @@ const { documentsPerPage } = require('../../projects/utils/pagination/documentsP
 const { getPagination } = require('../../projects/utils/pagination/pagination');
 const { getFilters } = require('./filters/get-filters');
 const { getProjectSearchSortByLinks } = require('./get-project-search-sort-by-links');
+const {
+	applications: { url: applicationsApiBaseUrl }
+} = require('../../../config');
 
 const getPageData = (query, applications, filters, pagination) => ({
 	...mapTitles('Projects', 'Project search'),
 	...getFilters(query, filters),
 	allProjectsSubNavigationRoutes,
 	applications: mapApplications(applications),
+	applicationsDownloadUrl: `${applicationsApiBaseUrl}/api/v1/applications/download`,
```

Comment:
just a heads up the api is not publicly accessible except on dev, so linking to this address will not work on test or prod.

the web-app will need to provide its own endpoint that forwards the request onto the api, in order to expose it to the browser. happy to help with this